### ISSUE TITLE
DarkMode- in guides the text is black needs to be white for contrast and accessibility 

### ISSUE THREAD

**Issue by GavinPahal** [Michelle2317/GoldiBite#138]
![IMG_0151](https://github.com/user-attachments/assets/9ed54682-32b7-4d9d-b629-4c91032e2e4f)
